src/domWrapper.ts

```diff
@@ -136,6 +136,14 @@ export class DOMWrapper<NodeType extends Node> extends BaseWrapper<NodeType> {
       this.trigger('input')
       // trigger `change` for `v-model.lazy`
       return this.trigger('change')
+    } else if (tagName === 'DIV' && this.attributes().contenteditable) {
```

Comment:
AFAIK contenteditable is available on all tags, not only on divs

and the presence of the attribute might not be enough, as I think in `<div contenteditable="true"><div id="inner"></div></div>`, `#inner` is also editable.

I think using the property `isContentEditable` might be more reliable